src/components/Form.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+function Form() {
+  return (
+    <div>
+      <form>
+        <input type="text" name="title" id="title" placeholder="Add Book Title" />
```

Comment:
- Hey, please, let's add an input for the author of the book as you've done for the title because what if the book and author is not available in your app and the user can't add the author except selecting from the list you've added. let's improve on this.